tests/validation/tests/v3_api/test_istio.py

```diff
@@ -47,17 +65,86 @@ def test_istio_resources():
     create_and_test_bookinfo_routing(app_client, app_ns, gateway_url)
 
 
-def create_and_verify_istio_app(p_client, ns, project, version):
+@if_test_rbac
+def test_rbac_istio_all_authenticated():
+    kiali_url, tracing_url, grafana_url, prometheus_url = get_urls()
+    cluster_owner = rbac_get_user_token_by_role(CLUSTER_OWNER)
+    cluster_member = rbac_get_user_token_by_role(CLUSTER_MEMBER)
+
     answers = {
-        "enableCRDs": "true",
-        "gateways.enabled": "true",
-        "gateways.istio-ingressgateway.type": "NodePort",
-        "gateways.istio-ingressgateway.ports[0].nodePort":
-            ISTIO_INGRESSGATEWAY_NODEPORT,
-        "gateways.istio-ingressgateway.ports[0].port": 80,
-        "gateways.istio-ingressgateway.ports[0].targetPort": 80,
-        "gateways.istio-ingressgateway.ports[0].name": "http2",
+        "global.members[0].kind": "Group",
+        "global.members[0].name": "system:authenticated",
+        "kiali.enabled": "true",
+        "tracing.enabled": "true",
+    }
+    update_istio_app(answers)
+
+    validate_access(kiali_url, cluster_owner)
+    validate_access(tracing_url, cluster_owner)
+    validate_access(grafana_url, cluster_owner)
+    validate_access(prometheus_url, cluster_owner)
+
+    validate_access(kiali_url, cluster_member)
+    validate_access(tracing_url, cluster_member)
+    validate_no_access(grafana_url, cluster_member)
+    validate_no_access(grafana_url, cluster_member)
+
```

Comment:
Also include test cases for enabling/disable Istio to be allowed only for global admin users.

Reply:
Added cases to try to disable and update istio using CLUSTER_MEMBER. Ignoring admin/owner use cases as we install and update istio in the tests using CLUSTER _OWNER token, so if there was an ApiError failure during the tests, we could safely assume it's due to something failing there and don't need an extra test case for it.